cadc-download-manager/src/main/resources/DataLinkServiceResolver.properties

```diff
@@ -1,29 +0,0 @@
-# default resourceID for unconfigured collections: caom2 workshop
```

Comment:
I think we should still have a configuration file just inside the `src` folder that has the comments as instructions with the name `DataLinkServiceResolver.properties`:
```
# Configuration for how cadc-download-manager resolves links to URLs
#
# Default resourceID for unconfigured collections: caom2 workshop
# ca.nrc.cadc.dlm.handlers.DataLinkServiceResolver.resourceID=ivo://cadc.nrc.ca/caom2ops

# This is a temporary short cut because we don't currently query an IVOA registry for
# the data collection and then find the appropriate DataLink capability
 
# <DataCollection resourceID> = <datalink service resourceID>
```

Reply:
Thanks. :)